src/informatica/azext_informatica/aaz/latest/informatica/data_management/organization/serverless_runtime/serverless/_resource_by_id.py

```diff
@@ -0,0 +1,365 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "informatica data-management organization serverless-runtime serverless resource-by-id",
```

Comment:
Why remove the `informatica data-management organization serverless-runtime show` command? If they do the same thing, I recommend you keep the `informatica data-management organization serverless-runtime show` command. But if you prefer this command, I recommend rename it by the following one as it didn't use any id argument when using this command.
```suggestion
    "informatica data-management organization serverless-runtime serverless show-resource",
```